libs/ngxtension/computed-from/src/computed-from.ts

```diff
@@ -59,53 +86,82 @@ export function computedFrom<Input extends object, Output = Input>(
  * }
  * ```
  */
-export function computedFrom(
-	sources: any,
-	operator?: OperatorFunction<any, any>,
-	injector?: Injector
-): Signal<any> {
+export function computedFrom<Input = any, Output = Input>(
+	...args: any[]
+): Signal<Output> {
+	const { normalizedSources, hasInitValue, operator, options } = _normalizeArgs<
+		Input,
+		Output
+	>(args);
+
+	let injector = options?.injector;
 	injector = assertInjector(computedFrom, injector);
+	/* try { //CUSTOM ERROR HANDLING FOR computedFrom */
+	//IF YOU PASS options.initialValue RETURN Signal<Output> WITHOUT ANY PROBLEM EVEN IF sources Observable ARE ASYNC (LATE EMIT) -> OUTPUT SIGNAL START EMITING SYNC initialValue!
```

Comment:
Can you convert the text to lowercase, please?

Reply:
Sorry forgot of this uppercase in the code - now I fixed it.
 @nartc & @eneajaho please let's me know if what you think of the new behavior I introduced?